lib/libcompiler_rt/Makefile

```diff
@@ -10,7 +10,7 @@ WARNS?=		2
 CFLAGS+=	${PICFLAG}
 CFLAGS+=	-fvisibility=hidden
 CFLAGS+=	-DVISIBILITY_HIDDEN
-.if (${MACHINE_ARCH} == "morello" || ${MACHINE_ARCH:Mmips*} || ${MACHINE_ARCH:Mriscv*c*})
+.if (${MACHINE_ARCH} == "aarch64c" || ${MACHINE_ARCH:Mmips*} || ${MACHINE_ARCH:Mriscv*c*})
```

Comment:
I wonder if we should be using that or `:Maarch64*c*`... there are no other forms, but... hmmmm
(same goes for a number of places in this PR)